R/utils.R

```diff
@@ -280,7 +280,10 @@ write_submission_file <- function(pred, forecast_reference_date, submission_dire
 #' Utility to get the reference date for a given date. This is the last day of
 #' the epiweek that the date falls in.
 get_forecast_reference_date <- function(date) {
-  MMWRweek::MMWRweek2Date(epiyear(date), epiweek(date)) + 6
+  if (!lubridate::is.Date(date)) {
+    date <- as.Date(date)
+  }
```

Comment:
Why not just 
```suggestion
    date <- as.Date(date)
```
It's idempotent